lib/base-compiler.js

```diff
@@ -202,20 +202,65 @@ class BaseCompiler {
         return options;
     }
 
-    getIncludeArguments(libraries) {
-        const includeFlag = this.compiler.includeFlag || "-I";
+    findLibVersion(selectedLib) {
+        const foundLib = _.find(this.compiler.libs, (o, libId) => {
```

Comment:
Just a note, this is fine, but can be simplified to `(o, libId) => libId === selectedLib.id` I think (losing the `{}` and `return` and the `()`s :) and below too.

Looks like that was how it was done before: we're all learning how to use modern JS :)